assembl/static2/css/components/navbar.scss

```diff
@@ -39,95 +37,201 @@
       text-decoration: none;
     }
 
+    &:after {
+      color: $navbar-active-text-color;
+      font-family: $font-family3;
+      text-decoration: none;
+      content: attr(data-text);
```

Comment:
This is used to render a `height: 0` element under the actual element to prevent uppercasing-on-hover to move the next elements due to the width changing.
Doing it this way allows to do this cosmetic feature with css only.